models/AuthBot.js

```diff
@@ -1,12 +1,18 @@
-"use strict";
+import mongoose from 'mongoose';
+import mockgoose from 'mockgoose';
 
-var MakeModel = require('common/makeModel');
+if (process.env.NODE_ENV === 'test') {
+  mockgoose(mongoose);
+}
 
-var AuthBotSchema = {
+const mongodbCredential = process.env.KINDLIZED_MONGODB;
+const db = mongoose.createConnection(mongodbCredential);
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).